nodogsplash/Makefile

```diff
@@ -59,6 +59,7 @@ define Package/nodogsplash/install
 	$(CP) $(PKG_BUILD_DIR)/openwrt/nodogsplash/files/etc/init.d/nodogsplash $(1)/etc/init.d/
 	$(CP) $(PKG_BUILD_DIR)/openwrt/nodogsplash/files/etc/uci-defaults/40_nodogsplash $(1)/etc/uci-defaults/
 	$(CP) $(PKG_BUILD_DIR)/openwrt/nodogsplash/files/usr/lib/nodogsplash/restart.sh $(1)/usr/lib/nodogsplash/
+	$(CP) $(PKG_BUILD_DIR)/forward_authentication_service/PreAuth/demo-preauth.sh $(1)/usr/lib/nodogsplash/login.sh
```

Comment:
why install a login.sh?

Reply:
See nodgogsplash v3.1.1 Changelog:

> nodogsplash (3.3.1)
> 
>   * Include preinstalled PreAuth Login script in package [bluewavenet]

The demo preauth script is now pre-installed and configured, but disabled in the NDS config. Very simple instructions on how to enable this are included in the config file itself. This is a fully functional script providing dynamic html to be served by libmicrohttpd.

It asks the client user for their name and email address, putting this information into a log file. Then a welcome page is displayed before continuing.

This is a ready to use alternative to the simple basic splash page that does not require a separate web server or script interpretor.

Users could of course create their own, more sophisticated scripts in ANY scripting language, or even use a compiled program if they so desired.